### PR TITLE
Fix the issue 2071

### PR DESCRIPTION
**Pull request by NipunaMarcus**
issue: https://github.com/ballerinalang/composer/issues/2071
and https://github.com/ballerinalang/composer/issues/2060

### CODE REVIEW

File: modules/web/js/ballerina/ast/default-ballerina-ast-factory.js [hunk 1]
```diff
@@ -254,14 +267,18 @@ DefaultBallerinaASTFactory.createAggregatedFunctionInvocationStatement = functio
...
-            return assignmentStmt;
+            const variableDefinitionStatement = BallerinaASTFactory.createVariableDefinitionStatement();
+            let leftOperandExpression = '';
+            args.functionDef.getReturnParams().forEach((returnParam, index) => {
+                if (index === 0) {
+                    leftOperandExpression = returnParam.type + ' ' + (returnParam.identifier
+                            ? returnParam.identifier : returnParam.type.substr(0, 1));
+                }
+            });
+            const expression = leftOperandExpression + ' = ' + functionInvokeString;
```
Comment (hemikak) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: You can use string templating here if needed IMO. Lets try and use it in the future.

File: modules/web/js/ballerina/ast/default-ballerina-ast-factory.js [hunk 2]
```diff
@@ -254,14 +267,18 @@ DefaultBallerinaASTFactory.createAggregatedFunctionInvocationStatement = functio
...
-            assignmentStmt.addChild(leftOp);
-            assignmentStmt.addChild(rightOp);
-            assignmentStmt.accept(new EnableDefaultWSVisitor());
-            return assignmentStmt;
+            const variableDefinitionStatement = BallerinaASTFactory.createVariableDefinitionStatement();
+            let leftOperandExpression = '';
+            args.functionDef.getReturnParams().forEach((returnParam, index) => {
+                if (index === 0) {
+                    leftOperandExpression = returnParam.type + ' ' + (returnParam.identifier
+                            ? returnParam.identifier : returnParam.type.substr(0, 1));
```
Comment (hemikak) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: Why/What are we substringing here ?

Comment (NipunaMarcus) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: we don't have a identifier to attach to the left operand expression so i attached the first letter of the type as the identifier.

File: modules/web/js/ballerina/ast/default-ballerina-ast-factory.js (hunk 1 above)
Comment (NipunaMarcus) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: sure will change

File: modules/web/js/ballerina/ast/default-ballerina-ast-factory.js (hunk 2 above)
Comment (hemikak) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: Ah right!

File: modules/web/js/ballerina/ast/default-ballerina-ast-factory.js (hunk 1 above)
Comment (NipunaMarcus) [inline modules/web/js/ballerina/ast/default-ballerina-ast-factory.js]: Can we change to string template later. Looks like it doesn't compile correctly.

**Pull request merged**